pkg/util/election_test.go

```diff
@@ -24,9 +24,9 @@ var (
 	electionInterval = flag.Duration("election-interval", 1*time.Second, "Scheduled election interval")
 )
 
-const extensionState = testhelpers.Timescale
-
 func TestPgLeaderLock(t *testing.T) {
+	// TODO (james): Replace hardcoded value
```

Comment:
should we move these into end to end tests?

Reply:
Hmmm... That would probably be a lot cleaner. Will take a look into doing that.

Reply:
It looks like `pkg/util/election.go` is dead code...

Reply:
I've removed this in https://github.com/timescale/promscale/pull/1253, waiting for one more reviewer then I'll rebase.